src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -81,6 +81,11 @@ public new CompressionLevelWithAuto RequestBodyCompressionLevel
         /// </summary>
         public bool AndroidNativeSupportEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Whether the SDK should add native support for Windows
+        /// </summary>
+        public bool WindowsNativeSupportEnabled { get; set; } = true;
```

Comment:
There is a test at `Sentry.Unity.Tests.ScriptableSentryUnityOptions` where we check the content of the actual `.asset` file so we don't accidentally drop any properties.

Reply:
added there.